.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: E2E
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  JavaAgentE2E:
+    name: Native Protocols(Management/JVM/Tracing/Meter/Event/Log/Profile/CDS)
```

Comment:
Is this name is too long?

<img width="940" alt="Screen Shot 2021-08-11 at 20 12 20" src="https://user-images.githubusercontent.com/15965696/129026467-a371a3bf-7008-421e-acfc-0e572d49ff2c.png">

Reply:
![image](https://user-images.githubusercontent.com/5441976/129030219-89dffa22-15a1-467a-ba7f-7ca30b628ea8.png)

It seems good in my screen(a 4K version).

Reply:
If we want to be shouter use `Native Management/JVM/Tracing/Meter/Event/Log/Profile/CDS`